src/fn/fn-table.scss

```diff
@@ -0,0 +1,175 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-table;
+$fn-table-item-margin: 0.375rem;
+$fn-table-item-focus-margin: 0.125rem;
+$fn-table-item-box-shadow: 0 0 0 $fn-table-item-focus-margin;
+
+.#{$block} {
+  @include fn-reset();
+
+  width: 100%;
+  max-width: 100%;
+  border-collapse: collapse;
+
+  &__row {
+    @include fn-reset();
+
+    @include fn-selected() {
+      .#{$block}__cell {
+        background: $fn-color-blue-2;
+        box-shadow: inset 0 -0.063rem 0 #0070f1;
+      }
+    }
+  }
+
+  .#{$block}__text {
+    @include fn-reset();
+    @include fn-ellipsis();
+
+    word-break: break-word;
+    white-space: normal;
+
+    &--no-wrap {
+      white-space: nowrap;
+    }
+  }
+
+  &__cell,
+  &__group-header,
+  &__header {
+    @include fn-reset();
+
+    box-shadow: inset 0 -0.063rem 0 #e4e5e7;
```

Comment:
This is a repeated value for List component. Could you move it to the variables file, please.